js/main.js

```diff
@@ -11,12 +12,14 @@ var $fullName = document.getElementById('fullName');
 var $location = document.getElementById('location');
 var $bio = document.querySelector('textarea');
 
-$saveButton.addEventListener('click', function (event) {
+$saveButton.addEventListener('submit', function (event) {
```

Comment:
You should attach this event listener to the form element, not the submit button :+1: